pkg/social/common.go

```diff
@@ -18,3 +21,30 @@ func isEmailAllowed(email string, allowedDomains []string) bool {
 
 	return valid
 }
+
+func jwtTokenScopes(token string) ([]string, error) {
+	chunks := strings.Split(token, ".")
+	if len(chunks) < 2 {
+		return nil, errors.New("malformed token")
+	}
+
+	rawData := chunks[1]
+	if l := len(rawData) % 4; l > 0 {
+		rawData += strings.Repeat("=", 4-l)
+	}
```

Comment:
Is there an universal standard for formating scopes this way in oauth2? given how google and github seems to do it differently. Afraid this will break existing users of the generic oauth implementation